packages/semcom-node/lib/component/services/component-controller.service.ts

```diff
@@ -10,26 +11,29 @@ export class ComponentControllerService implements ServerController {
       path: '/component',
       method: 'get',
       execute: (request) => this.all(request),
-    }
+    },
   ];
 
-  constructor(private components: ComponentService, private logger: LoggerService) { }
+  constructor(
+    private components: BaseComponentService,
+    private logger: LoggerService,
+  ) {}
 
   public async all(request: ServerRequest): Promise<ServerResponse> {
```

Comment:
The existing endpoint does not support querying of components. You will need to create a new endpoint (and route) to allow this.